aspnetcore/security/data-protection/extensibility/core-crypto.md

```diff
@@ -21,9 +21,9 @@ The **IAuthenticatedEncryptor** interface is the basic building block of the cry
 
 As its name suggests, the type is responsible for providing authenticated encryption and decryption services. It exposes the following two APIs.
 
-* Decrypt(ArraySegment<byte> ciphertext, ArraySegment<byte> additionalAuthenticatedData) : byte[]
+* Decrypt(ArraySegment\<byte> ciphertext, ArraySegment\<byte> additionalAuthenticatedData) : byte[]
```

Comment:
Let's code-fence these instead, since we want to avoid localization of them.

```suggestion
* `Decrypt(ArraySegment<byte> ciphertext, ArraySegment<byte> additionalAuthenticatedData) : byte[]`
```